docs/manual/guides/deployer.de.md

```diff
@@ -123,7 +125,7 @@ dann definiere diesen entsprechend, damit Deployer das auch weiß.
   }
 }
 ```
-
+{{% /notice %}}
```

Comment:
This would not work for Contao 4.9. I think this should be reverted?

Reply:
The guide is intended for Contao 4.13 and up though?

Reply:
> The guide is intended for Contao 4.13 and up though?

The reason for this is the usage of the migrate and backup commands.

Reply:
`contao:migrate` is already available in Contao 4.9.

Reply:
@richardhj I don't see usages of `contao:backup:*` in the recipe. Can I then safely remove the notice on top (about 4.13), and revert/update this change?

Reply:
Ah, no, it is about the maintenance tasks. Added a in Contao 4.13. See lines 70ff.

Reply:
So that means Contao 4.9 is irrelevant here @fritzmg.

Other (an advanced) option would be updating the recipe to detect Contao version and/or presence of `maintenance-mode` or Lexik bundle, and execute (or skip) the command accordingly.

Reply:
The guide only mentions `contao:maintenance:disable`. `contao:maintenance:enable` isn't even used. I think this line can be removed. `contao:maintenance:enable` is (mostly) not necessary anyway as the point of using Deployer are zero down time deployments. (Using `contao:maintenance:enable` would negate that.)

Reply:
> `contao:maintenance:enable` isn't even used

It is invoked by the recipe. `contao:maintenance:disable` is mentioned in context of failed deployment, when prod installation would be left hanging in maintenance-enabled state.

Reply:
> The guide only mentions `contao:maintenance:disable`. `contao:maintenance:enable` isn't even used. I think this line can be removed. `contao:maintenance:enable` is (mostly) not necessary anyway as the point of using Deployer is also zero down time deployments. (Using `contao:maintenance:enable` would negate that.)

However, the database is shared, so we must make sure no invalid transactions while migrating.

Reply:
Yes, it is meant (as far as I understand) as a mean of locking the database while `contao:migrate` is performed.

Reply:
I see. But then we should remove any mentions of zero down time in the guide (as there is now down time involved when using this recipe).

Reply:
Can I merge the pull request if I adjust the downtime text as follows?

_Die Idee hinter Deployer ist, dass es eine möglichst kurze Downtime bei Updates gibt._

_The idea behind Deployer is that there is the shortest possible downtime during updates._

Reply:
Yes, great suggestion! :)